src/test/java/Pages/PlaylistPage.java

```diff
@@ -0,0 +1,115 @@
+package Pages;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaylistPage extends BasePage{
+    public PlaylistPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    By myPlaylist = By.cssSelector("[class='playlist playlist']");
+    By deleteBtn = By.cssSelector("[title='Delete this playlist']");
+    By btnOk = By.cssSelector(".ok");
+    By btnPlus = By.cssSelector("[data-testid='sidebar-create-playlist-btn']");
+    By newPlaylistCreate = By.cssSelector("[data-testid='playlist-context-menu-create-simple']");
+    By playlistNameField = By.cssSelector(".create input");
+    By assertPlaylistNameIsInHeader = By.cssSelector("#playlistWrapper h1");
+    By playlistDeleteBtn = By.cssSelector(".btn-delete-playlist");
+    By listOfPlaylistNames = By.cssSelector("#playlists a");
+    By playlistActive = By.cssSelector(".playlist .active");
+    By playlistMenuEdit = By.cssSelector(".playlist .menu li ");
+    By playlistNameEditField = By.cssSelector("input[name='name']");
+    By assertRenamedPlaylistIsPresent = By.cssSelector(".playlist.playlist>a");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```